modules/Emails/Email.php

```diff
@@ -2695,8 +2695,16 @@ public function send()
             }
         }
 
+        $mailBefore = clone $mail;
+
         $mail = $this->setMailer($mail);
 
+        // use personal email by default
+
+        if($mailBefore->oe->type === 'user' && $mail->oe->type === 'system') {
```

Comment:
Do we need to consider the system_override?

Why mailBefore?  Why we checking the mail account before and after for?